packages/git/test/unit/lib/git.unit.test.ts

```diff
@@ -0,0 +1,88 @@
+'use strict'
+/* global beforeEach afterEach */
+
+import sinon from 'sinon'
+import {expect} from 'chai'
+import cp from 'child_process'
+import EventEmitter from 'events'
+import Git from '../../../src/git'
+
+describe('git', function () {
+  let mock: sinon.SinonMock
+  let git: Git
+  beforeEach(() => {
+    mock = sinon.mock(cp)
+    git = new Git()
+  })
+  afterEach(() => mock.restore())
+
+  it.skip('runs exec', function () {
```

Comment:
Just to be clear, these are the group of tests we're skipping for the time being correct?

Reply:
Yes. Just for the time being. I skipped them instead of deleting them so we still have a record of what we would like to test in the future.